auction-app-frontend/src/pages/TestPage.jsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react'
+
+const TestPage = () => {
+    const [data, setData] = useState();
+
+    useEffect(() => {
+        fetch('http://localhost:8080/test')
```

Comment:
We should move our URL for the server to file with constants and use that constant instead of raw string.